pkg/registry/servicecatalog/instance/storage.go

```diff
@@ -0,0 +1,44 @@
+package instance
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/rest"
+	genericregistry "k8s.io/kubernetes/pkg/registry/generic"
+	"k8s.io/kubernetes/pkg/runtime"
+)
+
+type serviceInstanceStorage struct {
+}
+
+// NewStorage creates a new rest.Storage responsible for accessing Instance
+// resources
+func NewStorage() rest.Storage {
```

Comment:
So, I would personally be fine if all this PR did was add a `NewStorage()` for each resource.  That way, we could parallelize the implementations of the storage for each resource and the experience of writing the rest storage piece for a resource could be shared by multiple people.